stac/geodes_sync.py

```diff
@@ -1,17 +1,22 @@
 import json
+import os
 import sys
 
 import requests
 import typer
+from prettytable import PrettyTable
 
 from airs.core.models.mapper import item_from_dict, to_json
-from airs.core.models.model import Asset, AssetFormat, Item, ItemFormat, Role
-from prettytable import PrettyTable
+from airs.core.models.model import (Asset, AssetFormat, Band, Item, ItemFormat,
+                                    Role)
 
 requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
 app = typer.Typer(add_completion=False, no_args_is_help=True)
 
 
+BANDS = ["B01", "B02", "B03", "B04", "B05", "B06", "B07", "B08", "B8A", "B09", "B10", "B11", "B12", "TCI"]
```

Comment:
could you move `BANDS = ["B01", "B02", "B03", "B04", "B05", "B06", "B07", "B08", "B8A", "B09", "B10", "B11", "B12", "TCI"]` in `airs/core/models/model.py`  and re-use it in `stac/geodes_sync.py` ?